backend/app/controllers/api/v1/accounts/project_developers_controller.rb

```diff
@@ -16,11 +15,23 @@ def create
           end
         end
 
+        def update
+          if current_user.role.to_sym != :project_developer
+            raise API::UnprocessableEntityError, I18n.t("errors.messages.user.no_project_developer")
+          end
+
+          current_user.account.update! account_params.except(:language)
```

Comment:
Language can be update only via Admin so I have move this out here

Reply:
you can add `before_filter :ensure_project_developer!` or `require_project_developer!` or maybe we should add authorization layer.

```
def ensure_project_developer!
  return if current_user.project_developer?   

  raise API::UnprocessableEntityError, I18n.t("errors.messages.user.no_project_developer")
end
```

Reply:
Like it, will do it ;)